Configuration/HLT/python/autoHLT.py

```diff
@@ -12,7 +12,8 @@
   'relval2018' : 'Fake2',
   'relval2022' : 'Fake2',
   'relval2023' : 'Fake2',
-  'relval2024' : 'GRun',
+  'relval2024' : 'Fake2',
+  'relval2025' : 'GRun',
   'relval2026' : '75e33',
```

Comment:
Hmm, 2026 is rather confusing by now as it is by now still phase1, rather than phase2 which should now be 2030?
When is that update redefining 2026 foreseen?

Reply:
Indeed, it is the next item on my todo list.

Reply:
See also https://github.com/cms-sw/cmssw/issues/46028